Dockerfile

```diff
@@ -492,8 +519,8 @@ RUN \
   export CXX_FOR_TARGET="${TARGET}-g++" ; \
   export CXX_FOR_${GOOS}_${GOARCH}="${TARGET}-g++" ; \
   export GOFLAGS="-mod=vendor" ; \
-  go install std && \
-  go install -buildmode=pie std
+  go install std cmd && \
+  go install -buildmode=pie std cmd
```

Comment:
I'd guess that this is a slight behavior change in that we'll now end up with the `-buildmode=pie` versions of `go` and `gofmt` in the install directory. But I'm also not sure that matters, especially if the builds are fine.